src/commands/general/caniuse.ts

```diff
@@ -0,0 +1,44 @@
+import CommandInt from "@Interfaces/CommandInt";
+import { MessageEmbed } from "discord.js";
+
+const caniuse: CommandInt = {
+  name: "caniuse",
+  description:
+    "Returns an up-to-date browser support table for a feature",
+  parameters: ["`<feature>`: the browser feature to define"],
+  run: async (message) => {
+    try {
+      const { channel, commandArguments } = message;
+
+      const feature = commandArguments.shift();
+      
+      if (!feature) {
+        await message.reply(
+          "Would you please try the command again, and provide the browser feature you want me to look for?"
+        );
+        await message.react(message.Becca.no);
+        return;
+      }
+      await channel.send(
+        new MessageEmbed()
+          .setTitle(`Caniuse: ${feature}`)
+          .setImage(`https://caniuse.bitsofco.de/image/${feature.replace(/ /g,'-')}.webp`)
```

Comment:
```suggestion
          .setImage(`https://caniuse.bitsofco.de/image/${feature}.webp`)
```
If we use `join` above with a `-` instead of spaces, we can skip the replace step here.

Reply:
thank you, i think that’s better